torch/csrc/autograd/init.cpp

```diff
@@ -54,6 +54,14 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
 
   m.def("_push_range", [](std::string name) { pushRange(std::move(name)); });
   m.def("_pop_range", []() { popRange(); });
+  m.def(
+      "_run_before_callbacks",
+      [](RecordFunction* rf, const std::string& funcName) {
```

Comment:
nit: why do you wrap this in a lambda and not just give `runBeforeCallbacks` directly?

Reply:
Was following the code above a little too closely. We can update this to `m.def("_run_before_callbacks", runBeforeCallbacks)`